README.md

```diff
@@ -1 +1,13 @@
-# solution-template-repository
\ No newline at end of file
+# Release Information

```

Comment:
Please create documentation using the standard format provided in https://github.com/fortinet-fortisoar/how-tos/tree/main/documenting